.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: CI Tests
+
+on: [push]
+
+jobs:
+  release:
```

Comment:
should this be named `build`? are we releasing?

Reply:
Good catch. I copied this across from the Sentry releases.